module/VuFind/src/VuFind/Search/Factory/SummonBackendFactory.php

```diff
@@ -50,7 +50,7 @@ class SummonBackendFactory extends AbstractBackendFactory
     /**
      * Logger.
      *
-     * @var \Laminas\Log\LoggerInterface
+     * @var \Psr\Log\LoggerInterface
```

Comment:
For now, this actually still is a Laminas logger -- we should change the comment back for the moment, and it can be adjusted to its final state in #4429.

```suggestion
     * @var \Laminas\Log\LoggerInterface
```